packages/sdk/src/types/TransactionExpiry.ts

```diff
@@ -28,9 +27,31 @@ class TransactionExpiry {
         public readonly expiryEpochSeconds: bigint
     ) {}
 
-    public toJSON(): number {
-        return Number(this.expiryEpochSeconds);
+    /**
+     * Get a string representation of the transaction expiry date in seconds since the Unix epoch.
+     * @returns {string} The string representation.
+     */
+    public toString(): string {
+        return this.expiryEpochSeconds.toString();
     }
+
+    /**
+     * Get a JSON-serializable representation of the transaction expiry date.
+     * @throws If the expiry is not representable as a number.
```

Comment:
Huh? Is this a mistake?

Reply:
I think I was at some point serializing as a `number`